README.md

```diff
@@ -132,6 +132,7 @@ Every model is written from scratch to maximize performance and remove layers of
 | Mixtral MoE | 8x7B | Mistral AI | [Mistral AI 2023](https://mistral.ai/news/mixtral-of-experts/)                                                                     |
 | Mistral | 7B, 123B | Mistral AI | [Mistral AI 2023](https://mistral.ai/news/announcing-mistral-7b/)                                                                  |
 | Nous-Hermes | 7B, 13B, 70B | NousResearch | [Org page](https://huggingface.co/NousResearch)                                                                          |
+| OLMo | 1B, 7B | Allen Institute for AI (AI2) | [OLMo project page](https://allenai.org/olmo)    |
```

Comment:
```suggestion
| OLMo | 1B, 7B | Allen Institute for AI (AI2) | [OLMo project page](https://arxiv.org/abs/2402.00838)    |
```

Let's use the OLMo paper here because it's hard to find any model information on the website.

Reply:
Gotcha
Also since the paper is accepted to ACL I'm pointing it to the paper uploaded on ACL Anthology - https://aclanthology.org/2024.acl-long.841/
I don't have any strong feelings eitherways but just thought this might be better